doc/en/user/source/production/config.rst

```diff
@@ -76,6 +76,14 @@ In some circumstances, you might want to completely disable the web administrati
 * Set the Java system property GEOSERVER_CONSOLE_DISABLED to true by adding -DGEOSERVER_CONSOLE_DISABLED=true to your container's JVM options
 * Remove all of the gs-web*-.jar files from WEB-INF/lib
 
+Disable the Auto-complete on web administration interface login 
+---------------------------------------------------------------
+
+In a production environment, it is a good practise to turn off Auto Complete on Web Admin login form.
```

Comment:
I disagree that is is a good practice to try and disable a secure browser behaviour and disable password autocomplete.

for reference also see eg:
- https://security.stackexchange.com/questions/49326/should-websites-be-allowed-to-disable-autocomplete-on-forms-or-fields
- https://bugzilla.mozilla.org/show_bug.cgi?id=425145#c55

Reply:
@mprins I see, will remove that part from the sentence. Will be glad to if you have any valuable suggestion as to what should the section describe in context what you have been pointed out.

Reply:
I would use similar wording as the paragraph above for the first sentence, eg.

> In some circumstances, you might want to disable auto-complete on the web admin login form.

and leave opinion out of it.

Reply:
done